### PR TITLE
Disable console logging for project-manager cli commands

### PR DESCRIPTION
**Pull request by 4e6**
### Pull Request Description

related: #10633

Changelog:
- fix: disable console logging for project manager CLI commands

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
Why? I found only this explanation:

> the project-manager --filesystem* commands can print error logs to stdout together with the json reply

So what?

File: lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/command/CommandHandler.scala [hunk 1]
```diff
@@ -29,5 +31,14 @@ final class CommandHandler(protocol: Protocol) {
         }
       )
       .map(_ => SuccessExitCode)
+  }
+
+  private def consoleLoggingOff: ZIO[ZAny, Throwable, Unit] =
+    ZIO.attempt {
+      val loggerSetup = LoggerSetup.get()
+      loggerSetup.teardown()
```
Comment (JaroslavTulach) [inline lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/command/CommandHandler.scala]: Disabling logging shouldn't be done by starting the logging  and that tearing it down, I think.

**Comment by 4e6** [comment]
> So what?

GUI expects output to be a JSON object and fails to parse it

**Comment by hubertp** [comment]
@4e6 I changed it slightly to avoid `teardown` call. All controlled via config file like in other places.

**Comment by 4e6** [comment]
@hubertp can confirm that it fixes the issue. Thank you!

**Review by hubertp — APPROVED** [review]

**Pull request merged**